### DISCUSSION TITLE
Reporting Security Vulnerabilities

### DISCUSSION THREAD

**Discussion by RichardHarperInfinityWorks** [jgraph/drawio#2557] [General]
Hello,

I work for a UK consultancy InfinityWorks and have access to a Static Code Analysis tool hcltechsw appscan, having run the tool against this project I have identified a few Security Vulnerabilities I wish to report.

I am happy to share the reports findings, and assist in remediating some of the issues if feasible.

Please let me know how to proceed, regards.

**Comment by davidjgraph** [comment]
Hi, please submit to support@diagrams.net. One issue per vuln would be great if possible.

**Reply by RichardHarperInfinityWorks** [reply]
Thanks @davidjgraph I will look to get them separated out and email them over as requested.

Once they are out I will notify you on here.

**Reply by RichardHarperInfinityWorks** [reply]
@davidjgraph just a heads-up that I am getting the findings in a descent format with the noise removed. I hope to get these out to you tomorrow.

**Reply by RichardHarperInfinityWorks** [reply]
@davidjgraph I have reported the 4 issues:

- DND-368
- DND-369 
- DND-370
- DND-371

**Reply by davidjgraph** [reply]
Thanks, acknowledge receipt. I agree these are all low priority so we will provide a resolution or an exception within 28 days.
